USERGUIDE.md

```diff
@@ -0,0 +1,168 @@
+# Requirements
+
+This module supports **iOS 11.0 and above**. It has been tested on iOS 11.0 and above.
+
+It is written in Swift 5.0 and can be used in compatible Xcode versions.
+
+In order to run your MiniApp you will have to provide the following,
+
+* MiniApp host application identifier (RASApplicationIdentifier)
+* Subscription key (RASProjectSubscriptionKey)
+* Base URL for API requests to the library (RMAAPIEndpoint)
+* Preference, if you want to make use of Test API Endpoints in your application or not
+
+
+# Configuration
+
+In your project configuration .plist you should add below Key/Value :
+
+| Key                          | Type    | Description                                                     | Optional| Default |
+| :----                        | :----:  | :----                                                           |:----:   |:----:   |
+| RASApplicationIdentifier     | String  | `Set your MiniApp host application identifier`                  |NO       |`none`   |
+| RASProjectSubscriptionKey    | String  | `Set your MiniApp subscription key`                             |NO       |`none`   |
+| RMAAPIEndpoint               | String  | `Provide your own Base URL for API requests`                    |NO       |`none`   |
+| RMAIsTestMode                | String  | `Loading mode of the API (`true`: testing, `false`: published)` |NO       |false    |
```

Comment:
`RMAIsTestMode` needs to be removed from the documentation until it's available on the backend. Also, the `Optional` column for this should be `YES`, shouldn't it?

Reply:
Removed `RMAIsTestMode` details from the documentation. And yes, the default value should be `true`